planner/core/physical_plans.go

```diff
@@ -223,6 +224,30 @@ func (p *PhysicalTableReader) ExtractCorrelatedCols() (corCols []*expression.Cor
 	return corCols
 }
 
+func (p *PhysicalTableReader) buildPlanTrace() *tracing.PlanTrace {
```

Comment:
Seems we should also consider `PhysicalIndexReader`, `PhysicalIndexLookUpReader`, `PhysicalIndexMergeReader`...

Reply:
we will finish other operator in future pr in order to keep each pr simple.